### DISCUSSION TITLE
Modelling using a GM4000QCI

### DISCUSSION THREAD

**Discussion by InFINNity-Deck** [mworion/MountWizzard4#346] [Q&A]
Dear Michel,

tonight I tried to do some modelling using a GM4000QCI mount (one of two at a Dutch university). The mount is controlled by an Eagle on which we installed Python 3.10 and the latest MW4. MW4 starts fine and connects to mount, camera and ASTAP (so three green buttons). However, the communication between MW4 and the mount is not going right. Pressing the Run-button initiates a slew, but then all action stops. Imaging is not initiated and the whole process stalls. We can get a single exposure through the image-pane, so the camera is well connected, but from within the modelling routine the camera is not activated (or the process stops before trying to do so). We also do not see a cursor in the hemisphere-pane, nor do we see the mount details (RA, DEC, etc) in the mount-window. Using the cursors we can control the mount, so some communication is present, but it seems one direction only.

The logfile of tonight's attempts is attached. We started in normal debug mode, but switched to trace later this evening in the hope to catch some more information for you.

[mw4-2024-02-12.zip](https://github.com/mworion/MountWizzard4/files/14251886/mw4-2024-02-12.zip)

Hopefully you can direct us in the right direction to get MW4 working properly with this GM4000QCI mount.

thanks in advance and
with kind regards,
Nicolàs

**Comment by mworion** [comment]
Hi Nicolas,
will have a look to this. It looks more than an issue with the camera. Could you tell which camera you are using ?
But still: QCI mount might have issues, but basic functions should work.
Michel

**Comment by InFINNity-Deck** [comment]
Hi Michel,

thanks for your response. The camera is a FLI proline 16803: https://www.flicamera.com/spec_sheets/archive/PL16803.pdf
The mount is running on FW 2.15.1, the latest QCI FW would be 2.17: https://www.10micron.com/forum/viewforum.php?f=4

We could upgrade the FW if that is required.

Nicolàs

**Comment by InFINNity-Deck** [comment]
Hi Michel,

got a new logfile from today, see attachment. Tomorrow an IT specialist is going to create a direct network between mount and Eagle. Hope t find out more then.
[mw4-2024-02-21.log.txt](https://github.com/mworion/MountWizzard4/files/14362214/mw4-2024-02-21.log.txt)

Nicolàs

**Comment by InFINNity-Deck** [comment]
Hi Michel,

isolating the Eagle and mount from the general network and connecting them directly did not solve the issue. Coming Friday I plan to visit the observatory and will try a FoxTerm connection as well as running my MountMonitor to see whether or not the mount responds, and if so, what it produces...

Nicolàs

**Comment by mworion** [comment]
Hi Nicolas,
whenever you have new information, please let me know.
Michel

**Comment by d33psky** [comment]
Old topic but I'm curious as well.

( groetjes :) )
